Specs/Scene/ModelSpec.js

```diff
@@ -1598,6 +1599,13 @@ defineSuite([
         });
     });
 
+    it('loads a glTF with WEB3D_quantized_attributes and accessor.normalized for higher precision decode matrix', function() {
```

Comment:
I'm not following the "higher precision decode matrix."  Why not add per-vertex normalized `ubyte8` colors to an existing model since that is the common use case?  In general, I would also like to avoid bloating the repo with a lot of new test models to test corner cases if we are able to augment existing ones reasonably.